treemodeladmin/tests/settings.py

```diff
@@ -35,33 +33,15 @@
     "wagtail.sites",
     "wagtail.users",
     "wagtail.contrib.styleguide",
+    "wagtail",
+    "wagtail.test.testapp",
```

Comment:
Does this project actually need testapp for any reason? I don't see any references to it in the tests anywhere. I think removing this also lets us remove the `WAGTAILADMIN_RICH_TEXT_EDITORS` definition below.

Including this also generates an error for me when I run `tox -e interactive`:

> django.db.migrations.exceptions.InconsistentMigrationHistory: Migration tests.0011_modelwithnullableparentalkey is applied before its dependency tests.0010_alter_customimage_file_and_more on database 'default'.

Reply:
Removed in 5f84cd0